src/spinnaker-dependencies.template

```diff
@@ -1,3 +1,4 @@
+# GENERATED FILE: ANY CHANGES WILL BE OVERWRITTEN
```

Comment:
want to also revert my changes that generate into and publish from the `build` directory?